account_analytic_distribution/models/account_move_line.py

```diff
@@ -0,0 +1,36 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import api, fields, models
+import json
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    first_analityc_account_id = fields.Many2one(
+        'account.analytic.account',
+        string="Analytic Account",
+        compute='_compute_first_analityc_account_id',
+        store=False,
+        index=True,
+        groups="analytic.group_analytic_accounting",
```

Comment:
If I'm not wrong, for original `analytic_distribution` field access is managed across `group` attribute in views, not at field definition. I'd prefer keep standard Odoo strategy here as well.